js/project-data.js

```diff
@@ -1,38 +1,128 @@
-  year:  '2015',
+  name: 'Tonic',
+  company: ['CANOPY', 'Back End Dev', '2015'],
   description: 'A daily selection of privately personalized reads; no accounts or sign-ups require',
-  tecnologies: ['html', 'css', 'javaScript'],
-  image:  'Snapshoot-Portfolio.png',
+  popupdescription: 'Lorem Ipsum is simply dummy text of the printing and or sign-ups requiretypes',
+  technologies: ['html', 'css', 'javaScript'],
+  image:  './img/Snapshoot-project3.png',
+  liveURL: 'https://osvaldobc.github.io/osvaldojose.github.io/',
+  sourceURL: 'https://github.com/OsvaldoBC/portfolio-setup',
 },
   {
-  Name: 'Multi-Post-Stories',
-  Company: 'CANOPY',
-  role:  'Back End Dev',
-  year:  '2015',
+  name: 'Multi-Post-Stories',
+  company: ['CANOPY', 'Back End Dev', '2015'],
   description: 'A daily selection of privately personalized reads; no accounts or sign-ups require',
-  tecnologies: ['html', 'css', 'javaScript'],
-  image:  'Snapshoot-Portfolio.png',
+  popupdescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lo',
+  technologies: ['html', 'css', 'javaScript'],
+  image:  './img/Snapshoot-project4.png',
+  liveURL: 'https://osvaldobc.github.io/osvaldojose.github.io/',
+  sourceURL: 'https://github.com/OsvaldoBC/portfolio-setup',
 },
 ];
-console.log(projectData);
+
+// fill dynamic mobile version"
+
+for (let i = 0; i < projectData.length; i++) {
+  console.log(projectData [i]);
+  let card1 = document.createElement('div');
+  card1.classList.add('container1');
+
+  let portfolio = document.createElement('div');
+  portfolio.classList.add('portfolio');
+
+  let contenido1 = document.createElement('div');
+  contenido1.classList.add('contenido1');
+
+  let cardh2 = document.createElement('h2');
+  cardh2.classList.add('tonic');
+  cardh2.textContent = projectData [i].name;
+  contenido1.appendChild(cardh2);
+
+  const buttons = document.createElement('ul');
+  buttons.classList.add('buttons');
+
+  for (let u = 0; u < projectData [i].technologies.length; u += 1) {
+    const tagLi = document.createElement('li');
+    tagLi.classList.add('int-button');
+    tagLi.textContent = projectData [i].technologies[u];
+
+    buttons.appendChild(tagLi);
+  }
+
+  const info = document.createElement('ul');
+  info.classList.add('info');
+
+  for (let u1 = 0; u1 < projectData [i].company.length; u1 += 1) {
+    const tagLi1 = document.createElement('li');
+    tagLi1.classList.add('tonic-detail2');
+    tagLi1.textContent = projectData [i].company[u1];
+    info.appendChild(tagLi1);
+  }
+
+  let cardp = document.createElement('p');
+  cardp.classList.add('project-detail');
+  cardp.textContent = projectData [i].description;
+
+  let image = document.createElement('img');
+  image.src = projectData [i].image;
+  image.alt = 'imagen';
+
+  const button = document.createElement('button');
+  button.classList.add('see');
+  button.setAttribute('type', 'button');
+  const butText = document.createTextNode('See Project');
```

Comment:
As requested by previous code reviewers your pop up design does'nt  match the figma design I'd suggest you style your pop-up both in your desktop version and mobile version of your web application according to the figma design. It is important to follow project requirements to avoid problems 

This is your pop-up design for you desktop:

![Screen 7](https://user-images.githubusercontent.com/79506105/189417320-0e297b1b-b747-4c10-955f-b4eeca952a0f.png)

This is your pop-up design for your mobile version:
![Screen 8](https://user-images.githubusercontent.com/79506105/189417473-cbb0d014-0461-4e4f-bd12-d82f507404d2.png)

Please your pop up should look like this: 

For your desktop is should look similar to this:
![Screen8](https://user-images.githubusercontent.com/79506105/189417777-429c99f2-26cb-4d59-b6c1-829c7f614f79.png)

And for your mobile pop-up it should look like this:

![Screen 9](https://user-images.githubusercontent.com/79506105/189417933-52a4a453-bc0c-4cba-9cae-4d46744345a0.png)

It would be nice to stick to the design on your figma template. Please pay attention to detail and inspect each and every feature in the template on figma .